### PR TITLE
Adding a Form in the Contact-Me Section with Validations

### PR DESCRIPTION
**Pull request by hernandanielzamora**
Adding a "form" tag in the contact me section with validations for every input tag. Also added a submit tag to create the button

### CODE REVIEW

**Comment by fpsapc** [comment]
I just add a button in get in touch section instead of input field and also add some transition to buttons so they looks well.

File: styles/styles.css [hunk 1]
```diff
@@ -463,7 +463,6 @@ div ul li {
 
 #getintouch {
   display: grid;
-  width: 100vw;
   height: 100vh;
   background-color: #f5f3f3;
   align-content: center;
```
Comment (Zuhaib042) [inline styles/styles.css]: - [x] Good job on the beautiful design but laying out your element in the form section should be with flex as in the requirements.
  -  In order to lay out the elements on the page you should use Flexbox.

File: index.html [hunk 2]
```diff
@@ -1,234 +1,253 @@
...
+    <form action="https://formspree.io/f/xoqzdnpa"
+    method="POST">
+      <section id="getintouch" class="border">
+        <h2 class="intouch">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch</h2>
+        <input class="input border" type="text" placeholder="Full name" maxlength="30" required>
+        <input class="input border" type="email" placeholder="Email address" required>
+        <textarea class="input" name="textarea" id="textarea" cols="30" rows="5" placeholder="Enter text here"
+          maxlength="500" required></textarea>
+          <div class="centerbtn"><button type="submit" class="getintouchbtn work-button">Get in touch</button>
```
Comment (Zuhaib042) [inline index.html:218]: - [x] Kindly consider applying the `pressed` state in button interactions as suggested in the figma design by using the `:active` pseudoclass as our design need to be similar to the figma design.
figma design
er

File: index.html [hunk 3]
```diff
@@ -1,234 +1,253 @@
...
+        <li>
+          <a href="https://twitter.com/salman-ahmadkhan1987" target="_blank" rel="noopener"> <i
+              class="fa-brands fa-twitter social"></i></a>
+        </li>
+        <li>
+          <a href="mailto:fpsapc@gmail.com" target="_blank" rel="noopener"> <i
+              class="fa-sharp fa-solid fa-m social"></i></a>
+        </li>
+      </ul>
+    </div>
```
Comment (Zuhaib042) [inline index.html]: - [x] Kindly consider using the social icons from the figma design as sticking to the figma design as much as possible is one of the project requirements.kindly consider downloading the figma design social icons to use.
figma design
fd

your design
as

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @hernandanielzamora  @fpsapc , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

## Highlights
✅ Correct Github flow


## Required Changes ♻️
- [x] kindly consider adding @hernandanielzamora as the second author in your readme as it will make your readme more professional.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fpsapc** [comment]
I also follow the buttons design in the last commit.

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @hernandanielzamora , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Required Changes ♻️
_Very well done on implementing all requested changes by previous reviewer Only one thing before you can move to next project , your footer is overlapping the form you need to fix this issue 

**your design:**
student contact form temp2

Please feel free to ask if you need help 😊

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @fpsapc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**